xskillscore/__init__.py

```diff
@@ -32,6 +32,7 @@
     rps,
     threshold_brier_score,
 )
+from .core.resampling import resample_iterations, resample_iterations_idx
```

Comment:
maybe the resampling functions dont need to get this top spot

Reply:
Maybe they can import as `xskillscore.resampling`. I imagine you could do:

```python
import .core.resampling as resampling
```

And then that would work globally as `xskillscore.resampling.resample_iterations` and so on